src/app/lists/page.tsx

```diff
@@ -7,18 +7,26 @@ import Title from "@/components/Title/Title";
 // disables caching
 export const revalidate = 0;
 
-const fetchData = async (): Promise<ListRow[]> => {
-    const { data } = await supabase.from("lists").select();
-    return data ?? [];
+interface Props {
+    data: ListRow[];
+    comment: string;
+}
+
+const fetchData = (): Promise<Props> => {
```

Comment:
fetchData should be async - and I think I prefer the await syntax.
in this case, await the Promise.all and save that to a const, then return the object you want 'normally'
(also, what happens in case of error from db?)

Reply:
Error handling was not implemented before I picked up the ticket, will sort on VFB-22. Have changed to await and async.

Reply:
I'd rather it be merged with 'ok' error handling which is then improved in VFB-22 - but I do think that the `?? []` does that fine for now!